pom.xml

```diff
@@ -319,7 +319,7 @@
             <dependency>
               <groupId>org.mockito</groupId>
               <artifactId>mockito-core</artifactId>
-              <version>2.28.2</version>
+              <version>3.6.0</version>
```

Comment:
nit: Is this change still needed?

Reply:
yes, for mocking static methods in `Table'

Reply:
I really don't feel good about mocking static methods, but OK.